<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Add">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>110</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodaj</string>
    </property>
   </widget>
   <widget class="QListWidget" name="TeamsList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>311</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Clear">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>390</y>
      <width>110</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Czyść</string>
    </property>
   </widget>
   <widget class="QComboBox" name="NumberofTeamsBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="NoTeamsLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>L. Drużyn : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="Draw">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>140</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>Losuj</string>
    </property>
   </widget>
   <widget class="QListWidget" name="GroupView1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>70</y>
      <width>281</width>
      <height>201</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="GroupView2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>320</y>
      <width>281</width>
      <height>201</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="GrBox1">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="GrBox2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>280</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Drużyny :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Save">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>450</y>
      <width>140</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>Zapisz</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
